src/FluffySpoon.AspNet.LetsEncrypt/LetsEncryptRenewalService.cs

```diff
@@ -65,7 +65,7 @@ public async Task StartAsync(CancellationToken cancellationToken)
 			foreach (var lifecycleHook in _lifecycleHooks)
 				await lifecycleHook.OnStartAsync();
 
-			_timer = new Timer(async (state) => await RunOnceAsync(), null, TimeSpan.Zero, TimeSpan.FromHours(1));
+			_timer = new Timer(async (state) => RunOnceWithErrorHandling(), null, TimeSpan.Zero, TimeSpan.FromHours(1));
```

Comment:
Await the call here. Also suffix the method name with Async.

Reply:
I'm embarssed I didn't catch that myself :)